Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
```

Comment:
It would be great to have some more documentation of the test that explains that it is for GUI regression (rather than checking the functionality/correctness of the bindings in general).

Reply:
I will add the comment to the multiple GUI files that use this code. There is a tradeoff between using the Context object (which was intended for interacting with a unique state that lives on the C++ side) and plain usage of the API. If all GUI calls go through Context then we make it into a facade and it becomes too "fat" and rather hard to use/maintain. For a while we've been moving in the other direction so that we're actually exercising the API as a scripter would do.